web/src/js/index.js

```diff
@@ -1,473 +1,479 @@
+        let localDate = logDate.toString().split(/ GMT/)[0];
+
+        row["key_escape"] = encodeURIComponent(row["key"]);
+        row["local_date"] = localDate;
+        return row;
+      });
+      setTimeout(function () {
+        $("#execution-log-list-table").DataTable({
+          dom: "rtp",
+          columnDefs: [
+            { orderable: false, targets: [0, 1, 2] },
+            { className: "dt-center", targets: [2] },
+          ],
+          order: [[0, "desc"]],
+        });
+      }, 10);
+      app.showExecutionLogListLoadingGif = false;
+    })
+    .catch((error) => {
+      iziToast.error({
+        color: "#EC2B55",
+        message: error,
+        messageColor: "white",
+        title: "Something went wrong",
+      });
+    });
+}
+
+// Get supported services
+function getServices() {
+  fetch(API_SERVICES, {
+    headers: {
+      "x-api-key": API_KEY,
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      app.serviceSettings = data["response"];
+
+      // get list of supported services
+      app.serviceList = Object.keys(data["response"]);
+
+      // convert settings to flat table
+      for (const service in data["response"]) {
+        for (resource in data["response"][service]) {
+          app.serviceSettingsFlat.push({
+            service: service,
+            resource: resource,
+            ttl: data["response"][service][resource]["ttl"],
+            enabled: data["response"][service][resource]["clean"],
+          });
+        }
+      }
+    })
+    .catch((error) => {
+      iziToast.error({
+        color: "#EC2B55",
+        message: error,
+        messageColor: "white",
+        title: "Something went wrong",
+        titleColor: "white",
+      });
+    });
+}
+
+// Get allowlist
+function getAllowlist() {
+  app.allowlist = [];
+  app.showAllowlistLoadingGif = true;
+  fetch(API_GET_ALLOWLIST, {
+    headers: {
+      "x-api-key": API_KEY,
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      let i = 1;
+      let allowlistRaw = data["response"]["allowlist"];
+
+      dayjs.extend(dayjs_plugin_utc);
+      dayjs.extend(dayjs_plugin_timezone);
+
+      app.allowlist = allowlistRaw.map((item) => {
+        // Parse Resource ID, i.e. split on ":"
+        let parsedResourceId = item["resource_id"].split(":", 2);
+        parsedResourceId.push(
+          item["resource_id"].slice(parsedResourceId.join("").length + 2)
+        );
+
+        let readableDate = dayjs.unix(item["expiration"]).tz(dayjs.tz.guess());
+
+        item["row_id"] = i++;
+        item["service"] = parsedResourceId[0];
+        item["resource"] = parsedResourceId[1];
+        item["id"] = parsedResourceId[2];
+
+        item["expiration_readable"] = readableDate.format(
+          "ddd MMM DD HH:mm:ss YYYY"
+        );
+
+        return item;
+      });
+
+      setTimeout(function () {
+        app.allowlistDataTables = $("#allowlist").DataTable({
```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/servian/aws-auto-cleanup/pullRequest?prid=9792602)
```suggestion
        app.allowlistDataTables = $("#allowlist").DataTable({
```